### ISSUE TITLE
Distal clavicle osteolysis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11010]
**Preferred term label:**
Distal clavicle osteolysis

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP clavicle and/or Zanca view of the AC joint x-ray showing osteolysis, cysts, erosions and resorption of the distal end of the clavicle

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal clavicle osteolysis (DCO)

**Your nano-attribution (ORCID)**
https://radiopaedia.org/articles/distal-clavicle-osteolysis?lang=us

**Comment by pnrobinson** [comment]
@MickeySegal what is the disease context?
There is a very old term that probably needs revision on our side:
[Progressive clavicular acroosteolysis HP:0000905](https://hpo.jax.org/browse/term/HP:0000905)
I am not sure we want to have "Progressive" in the term name. Is this the same item?

**Comment by MickeySegal** [comment]
The disease context is the entity " Distal clavicle osteolysis (DCO)".  

I will pass along your query to the orthopedist.

**Comment by MickeySegal** [comment]
The orthopedist responded:

I have not heard of or used the term "Progressive clavicular acroosteolysis" previously. After some research, it seems that distal clavicle osteolysis is a localized bone resorption (most common in athletes) that is limited to the distal end of the clavicle, while progressive clavicle acroosteolysis is part of a systemic, progressive bone resorption process that can involve the clavicle. It seems as though radiographs have a similar appearance, so as long as they ensure that the pathologies listed under progressive clavicle acroosteolysis make sense under DCO, I think it is reasonable to combine.

**Comment by pnrobinson** [comment]
There are now some annotations to the previous term.

The definition for here would be the following. 
TODO: see if this definition also applies to the hereditary diseases (e.g., https://pubmed.ncbi.nlm.nih.gov/21932319/). 

- label: Distal clavicle osteolysis
- parents: [Abnormal clavicle morphology (HP:0000889)](https://hpo.jax.org/app/browse/term/HP:0000889)
- pmid: PMID:35881753
- def: Osteolysis, cysts, erosions and resorption of the distal end of the clavicle, as documented by anteroposterior (AP) radiography of the clavicle and/or Zanca view (the AP X-ray beam is tilted by 10-15 degrees towards the head) of the acromioclavicular (AC) joint.

**Comment by pnrobinson** [comment]
@MickeySegal 
It seemed better to add a new term
label: Distal clavicular osteolysis
id: HP:0020321
def: Degeneration and resorption of the distal part of the clavicle near the acromioclavicular (AC) joint. Distal clavicular osteolysis (DCO) is an atypical injury occurring most often in athletes and weight lifters, specifically those who perform repetitive overhead movements. DCO results from repetitive excess load to the AC joint, typically through horizontal adduction, adduction, internal rotation, and forward/lateral flexion of the shoulder, such as bench pressing or overhead lifting. Imaging (AP clavicle and/or Zanca view of the AC joint) of DCO often demonstrates microcysts, loss of subchondral bone detail, and osteolysis of the distal clavicle.
